Author: Otto Kekäläinen <otto@seravo.fi>
Description: Extend timeout as some Debian pbuilder/sbuilder chroots may be very slow to run these

--- a/mysql-test/CMakeLists.txt
+++ b/mysql-test/CMakeLists.txt
@@ -64,7 +64,7 @@ ELSE()
 ENDIF()
 
 
-SET(MTR_FORCE perl ./mysql-test-run.pl --force)
+SET(MTR_FORCE perl ./mysql-test-run.pl --force --testcase-timeout=30 --suite-timeout=540)
 IF(EXISTS ${CMAKE_SOURCE_DIR}/mysql-test/suite/nist)
   SET(TEST_NIST ${MTR_FORCE} --comment=nist suite=nist ${EXP} && 
     ${MTR_FORCE} --comment=nist --force --suite=nist+ps ${EXP})
